src/iarc7_motion/iarc_tasks/track_roomba_task.py

```diff
@@ -162,8 +164,8 @@ def get_desired_command(self):
                 roomba_velocity = math.sqrt(roomba_x_velocity**2 + roomba_y_velocity**2)
 
                 # The overshoot is taking in the x velocity normalizing it and applying overshoot
-                x_overshoot = roomba_x_velocity/roomba_velocity * self._overshoot
-                y_overshoot = roomba_y_velocity/roomba_velocity * self._overshoot
+                x_overshoot = roomba_x_velocity/roomba_velocity * self._x_overshoot
```

Comment:
I don't think this will work because the unit vector being multiplied by is in the global frame while the x and y vectors are supposed to be in the roomba frame. If you think about x_overshoot and y_overshoot as a function of the roombas heading this gives you an ellipsoidal offset in the global frame. Meaning that if _x_overshoot != _y_overshoot you will get different overshoot amounts relative to the roomba depending on the roombas heading in the global frame.

I think you want to rotate the x and y overshoot vectors from the roomba frame to the global frame and then apply them to the positions in line 171 and 173.